src/classes/LD_LeadConvertOverride_CTRL.cls

```diff
@@ -361,7 +358,7 @@ public with sharing class LD_LeadConvertOverride_CTRL {
 	        lc.setLeadId(l.Id);
 	        if (!doNotCreateOpp && Schema.SObjectType.Opportunity.isCreateable()) lc.setOpportunityName(oppName);
 	        lc.setDoNotCreateOpportunity(!Schema.SObjectType.Opportunity.isCreateable() || doNotCreateOpp);
-	        lc.setConvertedStatus(leadConvStatus);
+	        lc.setConvertedStatus(l.Status);
```

Comment:
The default converted status will be set on the page, the user can change it if there are multiple options. Whatever is chosen will be used.